tests/dbshell/tests.py

```diff
@@ -70,4 +75,27 @@ def test_ssl_certificate_is_added(self):
             }))
 
     def get_command_line_arguments(self, connection_settings):
-        return DatabaseClient.settings_to_cmd_args(connection_settings)
+        return MySqlDatabaseClient.settings_to_cmd_args(connection_settings)
+
+
+class PostgreSqlDbshellCommandTestCase(SimpleTestCase):
```

Comment:
Could you put the new tests in a new file: `test_postgresql_psycopg2.py`? I'll rename the existing file to `test_mysql.py` in a separate commit.